svf-llvm/lib/LLVMModule.cpp

```diff
@@ -165,89 +165,38 @@ void LLVMModuleSet::build()
 void LLVMModuleSet::createSVFDataStructure()
 {
     getSVFType(IntegerType::getInt8Ty(getContext()));
-
+    Set<const Function*> candidateDecls;
+    Set<const Function*> candidateDefs;
     for (const Module& mod : modules)
     {
         /// Function
         for (const Function& func : mod.functions())
         {
-            SVFFunction* svfFunc = new SVFFunction(
-                getSVFType(func.getType()),
-                SVFUtil::cast<SVFFunctionType>(
-                    getSVFType(func.getFunctionType())),
-                func.isDeclaration(), LLVMUtil::isIntrinsicFun(&func),
-                func.hasAddressTaken(), func.isVarArg(), new SVFLoopAndDomInfo);
-            svfFunc->setName(func.getName().str());
-            svfModule->addFunctionSet(svfFunc);
-            addFunctionMap(&func, svfFunc);
-
-            for (const Argument& arg : func.args())
+            if (FunDefToDeclsMap.find(&func) != FunDefToDeclsMap.end() &&
```

Comment:
Add comment: "/// if this function func defined in ExtAPI but never used in application code (without any corresponding declared functions)."